packages/@sfcc-bff/basketapi/src/api/schema/basketResolvers.ts

```diff
@@ -440,34 +409,24 @@ export const basketResolver = (config: Config) => {
                 parameters: { couponCode: string },
                 context: AppContext,
             ) => {
-                try {
-                    const apiBasket = await addCouponToBasket(
-                        parameters.couponCode,
-                        config,
-                        context,
-                    );
-                    return new Basket(apiBasket);
-                } catch (e) {
-                    logger.error(`Error in basketResolvers(). ${e}`);
-                    throw e;
-                }
+                const apiBasket = await addCouponToBasket(
+                    parameters.couponCode,
+                    config,
+                    context,
+                );
+                return new Basket(apiBasket);
             },
             removeCouponFromBasket: async (
                 _: never,
                 parameters: { couponItemId: string },
                 context: AppContext,
             ) => {
-                try {
```

Comment:
These try catches are unnecessary, because they are essentially just catching and re-throwing. Apollo will already catch the error and create an `ApolloError`